firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsController.java

```diff
@@ -526,18 +521,18 @@ boolean finalizeSessions() {
     backgroundWorker.checkRunningOnThread();
 
     if (isHandlingException()) {
-      Logger.getLogger().d("Skipping session finalization because a crash has already occurred.");
+      Logger.getLogger().w("Skipping session finalization because a crash has already occurred.");
       return Boolean.FALSE;
     }
 
-    Logger.getLogger().d("Finalizing previously open sessions.");
+    Logger.getLogger().v("Finalizing previously open sessions.");
```

Comment:
I think the rest of this file are all great examples of where VERBOSE or WARN makes more sense than DEBUG. Nice!